qt/app.py

```diff
@@ -1182,11 +1182,26 @@ def updateTimeLine(self, refreshSnapshotsList = True):
                 item = self.timeLine.addSnapshot(sid)
             self.timeLine.checkSelection()
 
+    def validateOnTakeSnapshot(self):
+        hasMissing, missing = snapshots.hasMissing(self.config.include())
+        if hasMissing:
+            msgMissing = '\n'.join(missing)
+            msg = f'{_("The following folders are missing")}:\n\n{msgMissing}\n\n{_("Do you want to proceed?")}'
```

Comment:
This is hard for translators without knowing the context.

```suggestion
            msg = _('The following folders are missing: {folders} Do you want to proceed?'.format(
                folders=f'\n{msgMissing}\n\n')
```

I am aware that my suggestion looks ugly in code but it will support the translators.
- They do see the whole string at once.
- They don't see escaped newline characters they maybe do not understand. Translators are often not developers.
- The name of the placeholder variable `folders` gives them some more context.